aerogear/antora-ui
# add application security on dropdown and footer

## Motivation
https://issues.jboss.org/browse/AEROGEAR-8692

## What
Add Application Security to services

## Why
documentation

## How
add a drop down and footer link. 

## Verification Steps
- clone this repo
- checkout this branch
- cd ./atora-ui
- nvm use 8
- npm i
- npm i -g gulp-cli
- gulp preview
- access the preview at http://localhost:5252
- check the `Application Security` drop down and and the footer for the links 

![aerogear-docs](https://user-images.githubusercontent.com/16667688/57379142-320d1380-719e-11e9-84fc-91086f2bc07a.gif)

## Checklist:
